src/services.js

```diff
@@ -47,8 +47,11 @@ export function isOutsideOfBusinessHours(currentDatetime, holidays) {
  *  Transfer the incoming call to the overflow number
  */
 export async function transferCall(callObject) {
-  // Do nothing if the Webhook event is not associated to the main number:
-  if (callObject.number.id !== PHONE_NUMBER_AIRCALL_IDS.MAIN) {
+  // Do nothing if the Webhook event is not associated to the main number or
+  // the Toronto brokerage number:
+  if (!FORWARDED_AIRCALL_IDS.has(callObject.number.id)) {
+    logger.info('Call is for a number we use the answering service for.');
```

Comment:
Wouldn't this case be numbers that we _don't_ use the answering service for?

Either way the reference to "answering service" might be too specific (I could see us using this service to forward to other places), so maybe this could be something more like:
```suggestion
    logger.info('Ignoring call to an unknown number');
```

Reply:
They are not quite "unknown numbers" since they are still Aircall numbers we control, but maybe from the perspective of the service they are unknown. I like the direction of this though. Changing.